### PR TITLE
Adjust PR message stating how superflore was executed

### LINKED ISSUE #213: PR message stating how superflore was executed needs adjustment
In fact, it's not possible to reproduce the PR fully by running again as Superflore makes use of remote metadata. So state that instead of reproducing the PR, it'll list the command used originally.

### PR DESCRIPTION
**Pull request by andre-rosa**
- In fact, it's not possible to reproduce the PR fully by running again
    as superflore makes use of remote metadata. So state that instead of
    reproducing the PR, it'll list the command used originally.

Fixes #213 .

### CODE REVIEW

**Comment by andre-rosa** [comment]
@allenh1 << Ptal

**Comment by allenh1** [comment]
@andre-rosa I am going to let @tfoote decide on this.

I was okay with the way it was worded before, but you make a good point here. Tully is definitely better at wording things than I, so I'd like his input before merging this.

**Comment by andre-rosa** [comment]
@allenh1 @tfoote << Ptal :)

File: tests/test_utils.py [hunk 1]
```diff
@@ -176,12 +176,12 @@ def test_get_pr_text(self):
...
         tmp = sys.argv
         sys.argv = ['a', 'b', 'c', 'd']
-        expected = 'To reproduce this PR, run the following command.\n\n'\
+        expected = 'This pull request was generated by issuing:\n\n'\
                    '```\na b c d\n```\n'
         self.assertEqual(expected, get_pr_text())
         # test with an argument
         expected = 'sample\n'\
-                   'To reproduce this PR, run the following command.\n\n'\
+                   'This pull request was generated by issuing:\n\n'\
```
Comment (allenh1) [inline tests/test_utils.py]: @andre-rosa could you change this to "This pull request was generated by running the following command."?

Comment (tfoote) [reply tests/test_utils.py]: Yeah issuing is not a clear verb here.

```suggestion
                   'This pull request was generated by running the following command:\n\n'\
```

File: tests/test_utils.py [hunk 2]
```diff
@@ -176,12 +176,12 @@ def test_get_pr_text(self):
         """Test get PR text"""
         tmp = sys.argv
         sys.argv = ['a', 'b', 'c', 'd']
-        expected = 'To reproduce this PR, run the following command.\n\n'\
+        expected = 'This pull request was generated by issuing:\n\n'\
```
Comment (tfoote) [inline tests/test_utils.py]:
```suggestion
        expected = 'This pull request was generated by running the following command:\n\n'\
```

File: superflore/utils.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ def get_pr_text(comment=None):
     msg = ''
     if comment:
         msg += '%s\n' % comment
-    msg += 'To reproduce this PR, run the following command.\n\n'
+    msg += 'This pull request was generated by issuing:\n\n'
```
Comment (tfoote) [inline superflore/utils.py]:
```suggestion
    msg += 'This pull request was generated by running the following command:\n\n'
```

File: tests/test_utils.py (hunk 1 above)
Comment (andre-rosa) [reply tests/test_utils.py]: Thank you both! I had also to change the test case, done in latest patchset.

File: superflore/utils.py (hunk 3 above)
Comment (andre-rosa) [reply superflore/utils.py]: Just saw you had both files changed here after changing them. :) Thank you for that!

File: tests/test_utils.py [hunk 4]
```diff
@@ -173,12 +173,12 @@ def test_get_pr_text(self):
...
         tmp = sys.argv
         sys.argv = ['a', 'b', 'c', 'd']
-        expected = 'To reproduce this PR, run the following command.\n\n'\
+        expected = 'This pull request was generated by the following command:\n\n'\
                    '```\na b c d\n```\n'
         self.assertEqual(expected, get_pr_text())
         # test with an argument
         expected = 'sample\n'\
-                   'To reproduce this PR, run the following command.\n\n'\
+                   'This pull request was generated by issuing:\n\n'\
```
Comment (allenh1) [inline tests/test_utils.py]: @andre-rosa I think you missed one

File: superflore/utils.py [hunk 5]
```diff
@@ -47,7 +47,7 @@ def get_pr_text(comment=None, markup='```'):
     msg = ''
     if comment:
         msg += '%s\n' % comment
-    msg += 'To reproduce this PR, run the following command.\n\n'
+    msg += 'This pull request was generated by the following command:\n\n'
```
Comment (allenh1) [inline superflore/utils.py]: Sorry to be pedantic, but I'd really prefer `running the following command:`

Comment (andre-rosa) [reply superflore/utils.py]: No worries; done!

File: tests/test_utils.py (hunk 4 above)
Comment (andre-rosa) [reply tests/test_utils.py]: Done

**Pull request merged**